app/views/view_projects.py

```diff
@@ -0,0 +1,65 @@
+from flask import session, redirect, url_for, request, render_template, flash
+from app import app, forms
+from app.util import is_logged_in, get_user, error_message
+import pymesync
+import re
+
+
+@app.route('/projects/', methods=['GET', 'POST'])
+def view_projects():
+    if not is_logged_in():
+        return redirect(url_for('login', next=request.url_rule))
+
+    ts = pymesync.TimeSync(baseurl=app.config['TIMESYNC_URL'],
+                           test=app.config['TESTING'],
+                           token=session['token'])
+
+    user = get_user(session['user']['username'])
+    error_message(user)
+
+    projects = ts.get_projects()
```

Comment:
This `get_projects()` should be removed because you're also calling `get_projects()` below